dev/com.ibm.ws.ssl/test/com/ibm/ws/ssl/config/KeyStoreManagerTest.java

```diff
@@ -34,10 +35,27 @@ public void testAddWSKeyStoreToCache() throws Exception {
         assertNotNull(ksMgr.getKeyStore("testKeyStore"));
     }
 
+    /**
+     * Test to make sure getKeyStoreCount is working
+     */
+    @Test
+    public void testKeyStoreCount() throws Exception {
+        KeyStoreManager ksMgr = KeyStoreManager.getInstance();
+        int before = ksMgr.getKeyStoreCount();
+        WSKeyStore wsks = new WSKeyStore("unique");
+        ksMgr.addKeyStoreToMap("UniquetestKeyStore", wsks);
+        int after = ksMgr.getKeyStoreCount();
+        assertTrue("expect 1,2 but got: before " + before + " after: " + after, after - before == 1);
```

Comment:
Can also use the `assertEquals(String, Object, Object)` method. When it fails it will automatically use the String you provided as the failure message and then will add `"expected: <what-was-expected>; actual: <what-was-found>"`.